my_game.py

```diff
@@ -28,6 +28,7 @@
 PLAYER_MAX_SPEED = 7
 UFO_CHANGE_DIR_TIME_MAX = 10
 UFO_CHANGE_DIR_TIME_MIN = 2
+ASTEROIDS_TIMER = 5
```

Comment:
Let's call this ASTEROIDS_TIMER_SECONDS. Also, put in a comment which explains what it does.